python/ray/tests/test_actor_failures.py

```diff
@@ -514,62 +514,50 @@ def increase(self):
 )
 def test_multiple_actor_restart(ray_start_cluster_head):
```

Comment:
We don't promise the execution order for task retry. The following changes make the actor tasks idempotent.